cellprofiler/gui/editobjectsdlg.py

```diff
@@ -258,8 +258,8 @@ def build_ui(self):
         self.inside_print = False
 
         class CanvasPatch(matplotlib.backends.backend_wxagg.FigureCanvasWxAgg):
-            def __init__(self):
-                pass
+            def __init__(self, parent, id, figure):
+                matplotlib.backends.backend_wxagg.FigureCanvasWxAgg.__init__(self, parent, id, figure)
```

Comment:
Nice! Could you please pull this out into a separate pull request? It'll make issue tracking much easier. :)